### PR TITLE
Added test to verify behavior of tombstone records

### PR DESCRIPTION
**Pull request by sfc-gh-japatel**
SNOW-298877

Adding a test to repro behavior of tombstone records. 

Tombstone records are those records whose key is non null but value(Payload) is null. 

- We have decided that we would do a breaking change and **not** put an empty JSON(`{}`) inside table. 
- We can use SMT to filter empty records. 
  - https://kafka.apache.org/documentation/#connect_transforms

### CODE REVIEW

**Review by sfc-gh-zli — APPROVED** [review]
lgtm

File: src/test/java/com/snowflake/kafka/connector/internal/SinkServiceIT.java [hunk 1]
```diff
@@ -170,6 +172,68 @@ public void testIngestion() throws Exception {
...
+    LinkedList<String> contentResult = new LinkedList<>();
+    while (resultSet.next()) {
+      contentResult.add(resultSet.getString("RECORD_CONTENT"));
+    }
+    resultSet.close();
+
+    assert contentResult.size() == 1;
+
+    ObjectNode emptyNode = MAPPER.createObjectNode();
+    assert contentResult.get(0).equalsIgnoreCase(emptyNode.toString());
```
Comment (sfc-gh-rramachandran) [inline src/test/java/com/snowflake/kafka/connector/internal/SinkServiceIT.java:227]: assert that there is only one result too maybe?

**Review by sfc-gh-rramachandran — APPROVED** [review]
LGTM!

File: src/test/java/com/snowflake/kafka/connector/internal/SinkServiceIT.java (hunk 1 above)
Comment (sfc-gh-japatel) [reply src/test/java/com/snowflake/kafka/connector/internal/SinkServiceIT.java:227]: line 224

**Pull request merged**